tests/cypress/integration/remove_users_tasks_projects_organizations.js

```diff
@@ -17,69 +17,93 @@ describe('Delete users and tasks created during the test run.', () => {
                 password: Cypress.env('password'),
             },
         }).then(async (response) => {
-            authKey = await response['body']['key'];
+            authKey = await response.body.key;
```

Comment:
I am not sure ``await`` is necessary here

Reply:
Removed.